nss-day-cohort-54/exomine-group-project-venus-voyagers
# SelectFacility function built by team

Function built by whole team.


Fixes # 1

## Type of change

Please delete options that are not relevant.



# How Has This Been Tested?

1. select governor
2. select facility
3. choose mineral
4. see it populate under order 


# Checklist:
